src/core/scene/a-scene.js

```diff
@@ -169,7 +169,7 @@ var AScene = module.exports = registerElement('a-scene', {
         if (!camera || !canvas) { return; }
 
         // Update canvas.
-        if (!isMobile) {
+        if (!isMobile && !canvas.provided) {
```

Comment:
Though that is not specific, enough we could set `<canvas data-aframe-provided>` on the provided canvas